x-pack/test/security_solution_api_integration/test_suites/detections_response/utils/rules/prebuilt_rules/perform_upgrade_prebuilt_rules.ts

```diff
@@ -31,6 +31,7 @@ export const performUpgradePrebuiltRules = async (
     .set('kbn-xsrf', 'true')
     .set('elastic-api-version', '1')
     .set('x-elastic-internal-origin', 'foo')
+    .set('x-elastic-product-origin', 'kibana')
```

Comment:
These logs only surface in Kibana logs IIUC. I do not think we should exclude them from these logs but rather just as part of generating the "digest".

This enables us to still keep track of any deprecations we might need to fix in future.

Reply:
@jloleysens Do you mean we shouldn't add the header to `perform_upgrade_prebuild_rules`, and how do we exclude them while generating the "digest"?

Reply:
Yes, I meant for us to not add the header in a more widespread fashion and I'm not sure it makes sense in this case. Generally it is useful for us to see these logs, even when made by test code, as there may be something that needs to be fixed.